apps/web/src/pages/Applications/ApplicationResumeAddPage/components/AddExperienceForm.tsx

```diff
@@ -0,0 +1,206 @@
+          {intl.formatMessage({
+            defaultMessage: "Select a type of experience",
+            id: "jw6Umr",
+            description:
+              "Heading for the experience type section fo the experience form",
+          })}
+        </Heading>
+        <Select
+          label={intl.formatMessage({
+            defaultMessage: "Experience type",
+            id: "chnoRd",
+            description: "Label for the type of experience a user is creating",
+          })}
+          name="type"
+          id="type"
+          doNotSort
+          rules={{ required: intl.formatMessage(errorMessages.required) }}
+          nullSelection={intl.formatMessage({
+            defaultMessage: "Select a type",
+            id: "5PUycY",
+            description: "Default selection for the experience type field",
+          })}
+          options={[
+            {
+              value: "work",
+              label: intl.formatMessage(experienceTypeTitles.work),
+            },
+            {
+              value: "education",
+              label: intl.formatMessage(experienceTypeTitles.education),
+            },
+            {
+              value: "community",
+              label: intl.formatMessage(experienceTypeTitles.community),
+            },
+            {
+              value: "personal",
+              label: intl.formatMessage(experienceTypeTitles.personal),
+            },
+            {
+              value: "award",
+              label: intl.formatMessage(experienceTypeTitles.award),
+            },
+          ]}
+        />
+        <ExperienceDetails />
+        <TasksAndResponsibilities />
+        <Separator
+          orientation="horizontal"
+          decorative
+          data-h2-background="base(black.light)"
+          data-h2-margin="base(x2, 0)"
+        />
+        <div
+          data-h2-display="base(flex)"
+          data-h2-gap="base(x.25, x.5)"
+          data-h2-flex-wrap="base(wrap)"
+          data-h2-flex-direction="base(column) l-tablet(row)"
+          data-h2-align-items="base(flex-start) l-tablet(center)"
+        >
+          <Button
+            type="submit"
+            mode="solid"
+            value="return"
+            {...actionProps}
+            onClick={() => setValue("action", "return")}
+          >
+            {intl.formatMessage({
+              defaultMessage: "Save and go back",
+              id: "CuHYqt",
+              description: "Text for save button on profile form.",
+            })}
+          </Button>
+          <Button
+            type="submit"
+            mode="inline"
+            {...actionProps}
+            onClick={() => setValue("action", "add-another")}
+          >
+            {intl.formatMessage({
+              defaultMessage: "Save and add another",
+              id: "+7v9Dq",
+              description:
+                "Text for save button and add another button on experience form.",
+            })}
+          </Button>
+          <Link
```

Comment:
I'm getting this bad boy when using the "cancel and go back" button.  I'm OK if it's out of scope for this PR but let's make sure it's tracked on the board somewhere.
![image](https://user-images.githubusercontent.com/8978655/230165435-7317ffb6-2892-4df2-a793-b26f57cc3a1f.png)